Syntaxes/HTML (Django).sublime-syntax

```diff
@@ -0,0 +1,241 @@
+%YAML 1.2
+---
+# https://www.sublimetext.com/docs/syntax.html
+# https://html.spec.whatwg.org/multipage/syntax.html
+name: HTML (Django) Update
+scope: text.html.django
+version: 2
+
+extends: Packages/HTML/HTML.sublime-syntax
+
+file_extensions:
+  - html
+  - htm
+  - shtml
+  - xhtml
+
+# match on "{% load " or "{% extends "
+first_line_match: (?i){%\s?(?>load|extends)\s
+
+variables:
+  template_tags: "(
+        autoescape|endautoescape|\
+        block|endblock|\
+        csrf_token|\
+        cycle|\
+        debug|\
+        extends|\
+        filter|endfilter|\
+        firstof|\
+        for|empty|endfor|\
+        if|elif|else|endif|ifchanged|endifchanged|\
+        include|\
+        load|from|\
+        lorem|\
+        now|\
+        regroup|by|\
+        resetcycle|\
+        spaceless|endspaceless|\
+        templatetag|\
+        url|as|\
+        verbatim|endverbatim|\
+        widthratio|\
+        with|endwith|\
+        thumbnail|endthumbnail|\
+        static|get_static_prefix|get_media_prefix)"
+
+  template_filters: "(
```

Comment:
This is missing filters from https://docs.djangoproject.com/en/4.1/ref/contrib/humanize/